data/YashPimple.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "Yash Pimple",
   "type": "personal",
-  "bio": "Open Source Contributor | Web Developer | Blogger",
-  "tags": ["Open Source", "DevOps", "Kubernetes", "Docker", "React", "Node.js"],
+  "bio": "Open Source Contributor | DevOps | Cloud-Native | Tech-Blogger"
```

Comment:
```suggestion
  "bio": "Open Source Contributor | DevOps | Cloud-Native | Tech-Blogger",
```

Reply:
I have corrected the error made and update the JSON file